frontend/src/components/SpotGallery.tsx

```diff
@@ -1,29 +1,31 @@
 import {Spot} from "../models/Spot";
-import {Typography} from "@mui/material";
+import {Box, Container, Typography} from "@mui/material";
 import SpotCard from "./SpotCard";
+import './SpotGallery.css'
 
 type Props = {
     spots: Spot[]
-    deleteSpot: (id:string) => void
+    deleteSpot: (id: string) => void
 }
 
 export default function SpotGallery(props: Props) {
 
     return (
-        <div className="spot-gallery">
-            <Typography sx={{fontSize: "3rem", padding: "1rem"}} variant="h2" component="h2">
-                Spots
-            </Typography>
-            <ul>
-                {props.spots.map((spot) => {
-                    return (
-                        <SpotCard key={spot.id} spot={spot} deleteSpot={props.deleteSpot}/>
-                    )
-                })
-                }
-            </ul>
-
-        </div>
+        <Container className="spotgallery" maxWidth="lg">
+            <Box>
+                <Typography sx={{fontSize: "3rem", padding: "1rem"}} variant="h2" component="h2">
+                    Spots
+                </Typography>
+                <ul>
```

Comment:
Why use ul if you then dont actually end up using li? Why not make it e.g. a flexbox for a more responsive design?